examples/metatensor/example.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from metatensor import Labels, TensorBlock, TensorMap
+
+import sphericart
+import sphericart.metatensor
+
+
+l_max = 15
+n_samples = 100
+
+xyz = TensorMap(
+        keys=Labels.single(),
+        blocks=[
+            TensorBlock(
+                values=np.random.rand(n_samples, 3, 1),
+                samples=Labels(
```

Comment:
should we indicate somewhere (maybe here and in the docstring?) that the output will have the same samples as the input?